### ISSUE TITLE
twilio@4.0.1: decompress error

### ISSUE THREAD

**Issue by edinson0** [twilio/scoop-twilio-cli#10]
Failed to extract files from C:\Users\edins\scoop\apps\twilio\4.0.1\\twilio-win32-x64.tar.
Log file:
  ~\scoop\apps\twilio\4.0.1\\7zip.log

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/twilio/scoop-twilio-cli/issues/new?title=twilio%404.0.1%3a+decompress+error

**Comment by shrutiburman** [comment]
Hi @edinson0 , Thanks for reaching out. We've identified this issue internally and are working on it. Meanwhile, to unblock you can download the updated cli version from releases executables present [here](https://github.com/twilio/twilio-cli/releases/tag/4.0.1#:~:text=12%20days%20ago-,twilio%2D4.0.1.exe,-46.3%20MB).

**Comment by edinson0** [comment]
Hello, thank you very much, the problem has been solved.
